code/datums/components/foldable.dm

```diff
@@ -80,6 +80,9 @@ TYPEINFO(/datum/component/foldable)
 	throw_speed = 1
 	throw_range = 4
 	w_class = W_CLASS_BULKY
+	stamina_damage = 40
+	stamina_cost = 17
+	stamina_crit_chance = 10
```

Comment:
not sure what's going on with this unlisted buff to briefcases
```suggestion
```

Reply:
I replied to this later down, and just for sake of keeping this more clean I'm going to close this one.